app/Allocator/Stand/AirlineTerminalArrivalStandAllocator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Allocator\Stand;
+
+use App\Models\Vatsim\NetworkAircraft;
+use App\Services\AirlineService;
+use Illuminate\Database\Eloquent\Collection;
+
+class AirlineTerminalArrivalStandAllocator extends AbstractArrivalStandAllocator
+{
+    /**
+     * @var AirlineService
+     */
+    private $airlineService;
+
+    public function __construct(AirlineService $airlineService)
+    {
+        $this->airlineService = $airlineService;
+    }
+
+    protected function getPossibleStands(NetworkAircraft $aircraft): Collection
+    {
+        if (($airline = $this->airlineService->getAirlineForAircraft($aircraft)) === null) {
```

Comment:
Is there a cleaner way to check this?

Reply:
Have split the assignment out to previous line to make clearer :)